ui/vendor/css/bootstrap.css

```diff
@@ -264,7 +264,7 @@ th {
   font-family: 'Glyphicons Halflings';
 
   src: url('../fonts/glyphicons-halflings-regular.eot');
-  src: url('../fonts/glyphicons-halflings-regular.eot?#iefix') format('embedded-opentype'), url('../fonts/glyphicons-halflings-regular.woff') format('woff'), url('../fonts/glyphicons-halflings-regular.ttf') format('truetype'), url('../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular') format('svg');
+  src: url('../fonts/glyphicons-halflings-regular.eot?#iefix') format('embedded-opentype'), url('../fonts/glyphicons-halflings-regular.woff2') format('woff2'), url('../fonts/glyphicons-halflings-regular.woff') format('woff'), url('../fonts/glyphicons-halflings-regular.ttf') format('truetype'), url('../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular') format('svg');
```

Comment:
I think we need to add the `glyphicons-halflings-regular.woff2` resource, per console
https://github.com/mozilla/treeherder/blob/master/ui/vendor/fonts/

![glyphiconerrors](https://cloud.githubusercontent.com/assets/3660661/8856700/658b774a-313a-11e5-9c26-4f59390a0bfa.jpg)